src/app/shared/user-details/user-details.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="container">
+    <div>{{ user?.firstName + " " + user?.lastName }}</div>
+    <div>{{ user?.isDeleted ? 'Deleted' : 'Active' }}</div>
+    <div>Login Id: {{ user?.login }}</div>
+    <div>First Name: {{ user?.firstName }}</div>
+    <div>Last Name: {{ user?.lastName }}</div>
+    <div>Age: {{user?.age}}</div>
```

Comment:
Guess there is styling left out here. Adding a comment so that you can have a track of it